content/documentation/staging/faq/authentication/x-forwarded-port.adoc

```diff
@@ -0,0 +1,27 @@
+---
+title: How to forward OpenID login to Kiali behind a Virtual Service
```

Comment:
I think we want a more generic focus, as this is not specific to VirtualServices. Something like:

```suggestion
title: How to configure the originating port when Kiali is served behind a proxy (OpenID support)
```

but rephrase as you see better.